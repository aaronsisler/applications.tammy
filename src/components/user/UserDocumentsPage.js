import React from 'react';
import UserDocumentsContainer from './UserDocumentsContainer';

export default class UserDocumentsPage extends React.Component {
    constructor(props) {
        super(props);
    }

    render() {
        return (
            <div id="user_documents_page">
                <UserDocumentsContainer />
            </div>
        );
    }
}
