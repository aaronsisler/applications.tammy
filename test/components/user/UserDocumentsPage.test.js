import React from 'react';
import { shallow } from 'enzyme';
import UserDocumentsPage from 'User/UserDocumentsPage';

describe('UserDocumentsPage', () => {
    it('should render UserDocumentsPage correctly', () => {
        const wrapper = shallow(<UserDocumentsPage />);
        expect(wrapper).toMatchSnapshot();
    });
});
