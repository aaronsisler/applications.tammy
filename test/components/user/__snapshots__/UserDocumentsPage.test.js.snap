// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserDocumentsPage should render UserDocumentsPage correctly 1`] = `
<div
  id="user_documents_page"
>
  <Connect(UserDocumentsContainer) />
</div>
`;
